src/main/java/hudson/scm/SubversionSCM.java

```diff
@@ -1392,7 +1392,13 @@ else if (project.getLastBuild()!=null) {
 
         final Map<String,ISVNAuthenticationProvider> authProviders = new LinkedHashMap<String,
                 ISVNAuthenticationProvider>();
-        for (ModuleLocation loc: getLocations()) {
+
+        EnvVars env = null;
+        if (project.getLastCompletedBuild() != null) {
```

Comment:
@oleg-nenashev Before to do any modifications, I need to understand what do you mean. What static analysis tool?